ee/client/omnichannel/additionalForms/DepartmentForwarding.js

```diff
@@ -6,7 +6,10 @@ import { useEndpointData } from '../../../../client/hooks/useEndpointData';
 
 export const DepartmentForwarding = ({ departmentId, value, handler, label, placeholder }) => {
 	const t = useTranslation();
-	const { value: data } = useEndpointData('livechat/department');
+	const onlyMyDepartments = true;
+	const param = useMemo(() => ({ onlyMyDepartments }), [onlyMyDepartments]);
+
+	const { value: data } = useEndpointData('livechat/department', param);
```

Comment:
you could use `const param = { onlyMyDepartments: true }; ` declaring at the file scope, so you can "save" one hook